net/nginx/files/nginx.init

```diff
@@ -75,6 +75,54 @@ nginx_init() {
 	logger -t "nginx_init" -p "daemon.info" "using ${CONF} (the test is ok)"
 }
 
+add_mdns() {
+        local cfg="$1"
+        local port enabled service
+
+        config_get enabled "$cfg" enabled
+        config_get port "$cfg" listen
+
+        port=$(echo "$port" | head -n1 | awk '{print $1;}')
+
+
+        if [ "${enabled}" != "0" ] && [ "${enabled}" != "false" ] && [ -n "${port}" ]; then
+		case "${port}" in
+			"80")
+				service="http"
+				;;
+			"8080")
+				service="http-alt"
+				;;
+			"443")
+				service="https"
+				;;
+			"8443")
+				service="pcsync-https"
+				;;
+			*)
+				service="custom-http-${port}"
+				;;
+		esac
+
+		procd_add_mdns_service "$service" "tcp" "$port" "daemon=nginx"
+        fi
+}
+
+configure_mdns() {
+	local mdns="$(uci -q get nginx.global.mdns)"
+
+        if [ "${mdns}" = "1" ] || [ "${mdns}" = "true" ]; then
```

Comment:
Nice mix of styles here. `[ ... ]` is basically `if`. Not important. 

It would improve readability to tab indent here inside the if block. Indent equally, using tabs.
So here, maybe outdent the `if` and `fi` to match `local`.

Reply:
Hello,

Fixed the indentation.